crates/sui/src/ethereum_client.rs

```diff
@@ -20,6 +20,11 @@ use crate::serialize_or_execute;
 
 pub const ETH_DWALLET_MODULE_NAME: &IdentStr = ident_str!("eth_dwallet");
 pub const CREATE_ETH_DWALLET_CAP_FUNC_NAME: &IdentStr = ident_str!("create_eth_dwallet_cap");
+pub const INIT_STATE_FUNC_NAME: &IdentStr = ident_str!("init_state");
+pub const ETHEREUM_STATE_MODULE_NAME: &IdentStr = ident_str!("ethereum_state");
+pub const LATEST_ETH_STATE_STRUCT_NAME: &IdentStr = ident_str!("LatestEthereumState");
```

Comment:
After we add the move modules, I think these should move to where we implement the rust versions of the move objects we use(inside sui-types).